config/checkstyle-checks.xml

```diff
@@ -523,6 +523,14 @@
       -->
       <property name="severity" value="ignore"/>
     </module>
+    <module name="MissingNullCaseInSwitch">
+      <!--
+        We will not use this check because case null
+        is only supported by newer versions of Java (21).
+        Using a 'case null' will not compile in older Java code.
```

Comment:
Examples in issue show violations only show up when using `->` in the case. Aren't we good to enable this since we only use `:` cases currently? What earliest Java version is `->` cases supported?

Reply:
I see regression and tests have `:`, so it was slightly misleading to me from the issue. I will resolve this.